src/backend/database_models/tool.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
```

Comment:
what was the original goal of having tools definitions in the DB?

not really sure how we would use it. also for something like the `implemenation` column, the current situation is that `.implemenation` is referencing the tool class itself and can call on its methods. not sure how it would work here.

Reply:
let's discuss it